tests/robot-tests/tests/general_public/redirects.robot

```diff
@@ -45,19 +55,20 @@ Verify that routes without an absolute path still permit trailing slashes
 
 Verify that routes with /1000 are redirected without them
     user navigates to public frontend    %{PUBLIC_URL}/data-catalogue/1000
-    user waits until page contains    Browse our open data
-    user checks url equals    %{PUBLIC_URL}/data-catalogue
+    user waits until page contains    Data catalogue
+    user checks removed auth from url contains    ${PUBLIC_URL_WITHOUT_AUTH}/data-catalogue
 
 Verify that routes with search parameters retain them
     user navigates to public frontend    %{PUBLIC_URL}/data-catalogue?foo=bar&baz=zod
-    user waits until page contains    Browse our open data
-    user checks url equals    %{PUBLIC_URL}/data-catalogue?foo=bar&baz=zod
+    user waits until page contains    Data catalogue
+    user checks removed auth from url contains    ${PUBLIC_URL_WITHOUT_AUTH}/data-catalogue
 
 Verify that multiple rules work together
     user navigates to public frontend    %{PUBLIC_URL}/data-catalogue/1000?foo=bar&baz=zod
-    user waits until page contains    Browse our open data
-    user checks url equals    %{PUBLIC_URL}/data-catalogue?foo=bar&baz=zod
+    user waits until page contains    Data catalogue
+    user checks removed auth from url contains   ${PUBLIC_URL_WITHOUT_AUTH}/data-catalogue
 
     user navigates to public frontend    %{PUBLIC_URL}/data-catalogue/1000/?foo=bar
-    user waits until page contains    Browse our open data
-    user checks url equals    %{PUBLIC_URL}/data-catalogue?foo=bar
+    user waits until page contains    Data catalogue
+    user checks removed auth from url contains    ${PUBLIC_URL_WITHOUT_AUTH}/data-catalogue
```

Comment:
A suggestion to edit the wording of the keyword like so
'user checks url without auth equals'